src/styles/main.scss

```diff
@@ -1,3 +1,261 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+$color-card-border: #f3f3f3;
```

Comment:
Please move to separate file
![image](https://user-images.githubusercontent.com/54936044/225754219-2421778e-e033-4c6e-b746-46cff9ebaa23.png)